[tech] add support to use the existing dust context

hegdeashwin opened the pull request:
## Changes

- [x] Add support to use the existing dust context for Node v8 or above.

- [x] All test are working locally, no change in test cases.

- [x] Change package version from `0.0.1` to `0.0.2`.

References:
https://github.com/linkedin/dustjs-helpers/pull/146

Commit:
[tech] add support to use the existing dust context

gabrielcsapo reviewed (approved):
Can we update the docs? I know there non but this would be a good opportunity to start! Other than that looks good to me!

grawk commented:
backwards compat?

gabrielcsapo commented:
@grawk going to agree with you, this is not backwards compatible.

Commit:
[tech] return dust to pass the same dust references

hegdeashwin commented:
## Updates

- [x] Changed the solution approach as per linkedin/dustjs-helpers#146 with an addition to Return dust object, to maintain the same references. It should be backward compatible now. Tested under Node v8.9.3 and v6.12.2 and is working.

- [x] All test are working locally, no change in test cases.

- [x] Added possible references to helpers document README.md

kumarrishav commented:
LGTM now.  👍  

@grawk  @gabrielcsapo

grawk commented:
Thanks! :+1:

grawk commented:
Published as `pulvus-provide@1.0.3`
